plugins/ScheduledReports/templates/index.twig

```diff
@@ -9,19 +9,40 @@
 
 {% block content %}
 
-<div class="emailReports" piwik-manage-scheduled-report>
-
-    <span id="reportSentSuccess"></span>
-    <span id="reportUpdatedSuccess"></span>
-
-    <div>
-        {% import 'ajaxMacros.twig' as ajax %}
-        {{ ajax.errorDiv() }}
-        {{ ajax.loadingDiv() }}
-        {% include "@ScheduledReports/_listReports.twig" %}
-        {% include "@ScheduledReports/_addReport.twig" %}
-        <a id='bottom'></a>
-    </div>
+{% set reportParametersScheduledReportsEvent %}{{ postEvent('Template.reportParametersScheduledReports') }}{% endset %}
+
+{# load the Template.reportParametersScheduledReports event twice, once here outside of vue so any inline scripts
+   will execute. then again in the vue slot so it will be used correctly. This is hack to provide some level of BC. #}
+<div style="display:none" vue-entry-ignore ng-non-bindable>
+    {{ reportParametersScheduledReportsEvent|raw }}
+</div>
+
+<div
+  vue-entry="ScheduledReports.ManageScheduledReport"
+  title="{{ title|default(null)|json_encode|e('html_attr') }}"
```

Comment:
Using an attribute named `title` causes a tooltip to be shown everywhere within that div. So might be better to use another name here.

Reply:
In addition the `json_encode` here also causes special charts like umlauts to be displayed encoded in the tooltip.
Does Vue always perform a json_decode on input attributes? Or might that happen for other parameters within vue as well?